babel.config.json

```diff
@@ -7,17 +7,12 @@
       "@babel/preset-env",
       {
         "targets": "> 1%, not dead, Chrome 86, ChromeAndroid 86, Edge 86, Firefox 84, FirefoxAndroid 84, Firefox ESR, iOS 13, Safari 13",
-        "modules": "auto",
-        "useBuiltIns": false,
```

Comment:
I'm not sure if theses options should be changed.

Reply:
These options are default values, we can remove it won't be changed.